houdini.sql

```diff
@@ -1465,6 +1465,7 @@ CREATE TABLE penguin_stamp (
   penguin_id INT NOT NULL,
   stamp_id INT NOT NULL,
   recent BOOLEAN NOT NULL DEFAULT TRUE,
+  in_game_session BOOLEAN NOT NULL DEFAULT TRUE,
```

Comment:
Is this required to be in the database, or could you also use Redis for this?
I am just considering this, because there is a good chance that people will be confused about the migrations.

Reply:
Honestly you got a point

Reply:
I will try moving it to redis. It is probably better since this information is not meant to last a long amount of time